features/poller/api/src/main/java/org/opennms/netmgt/poller/PollStatus.java

```diff
@@ -69,6 +69,8 @@ public class PollStatus implements Serializable {
     private final Map<String, Number> m_properties = Collections.synchronizedMap(new LinkedHashMap<String, Number>());
 
     private DeviceConfig deviceConfig;
+
+    private String dcbScriptDebugOutput;
```

Comment:
Can we make this part of the `DeviceConfig` class? This would prevent cluttering this API even more.